justfile

```
@@ -264,11 +264,14 @@ linkerd-tag := `bin/root-tag`
 
 docker-arch := ''
 
-_pause-image := `yq .gateway.pauseImage multicluster/charts/linkerd-multicluster/values.yaml`
-
 _pause-load: _k3d-init
-    if [ -z "$(docker image ls -q '{{ _pause-image }}')" ]; then docker pull -q '{{ _pause-image }}'; fi
-    k3d image import --mode=direct --cluster='{{ k3d-name }}' {{ _pause-image }}
+    #!/usr/bin/env bash
+    set -euo pipefail
+    pauseImage="$(yq .gateway.pauseImage multicluster/charts/linkerd-multicluster/values.yaml)"
+    if [ -z "$(docker image ls -q $pauseImage)" ]; then
+       docker pull -q "$pauseImage"
+    fi
+    k3d image import --mode=direct --cluster='{{ k3d-name }}' "$pauseImage"
```

TIOLI: temp variable with short scope should get short name

```suggestion
    img="$(yq .gateway.pauseImage multicluster/charts/linkerd-multicluster/values.yaml)"
    if [ -z "$(docker image ls -q "$img")" ]; then
       docker pull -q "$img"
    fi
    k3d image import --mode=direct --cluster='{{ k3d-name }}' "$img"
```

Plus, I think we technically should quote this in the subshell..